src/v3/src/mocks/browser.ts

```diff
@@ -19,6 +19,19 @@ import { loadScenario } from './registry';
 
 const DEFAULT_SCENARIO_NAME = 'development';
 
+const getMockResponse = (mockResponseParam: string | null): any | undefined => {
+  if (!mockResponseParam) {
+    return undefined;
+  }
+
+  // Enables us to feed v2 mock files to be loaded as a mock response
+  // i.e. '@okta/mocks/data/idp/idx/identify.json'
+  if (mockResponseParam.includes('@okta/mocks')) {
+    return require(mockResponseParam);
+  }
+  return require(`./response${mockResponseParam}.json`);
+};
```

Comment:
Added this function to allow us to re-use mock files in v2 folder (this allows us to test against a11y for different views without having to duplicate the mock file in v3.

Reply:
This ended up not working in e2e test :(